modules/microshift-get-mirror-reg-container-image-list.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-get-mirror-reg-container-image-list_{context}"]
+= Getting the {product-title} mirror registry container image list
+
+To use a mirror registry, you must know where the container image list is stored. The list of the container image references used by a specific version of {product-title} is provided in the `release-<arch>.json` files that are part of the `microshift-release-info` RPM package.
+
+.Procedure
+
+. Access the list of container image references by using one of the following methods:
+
+.. If the package is installed on a {product-title} host, access the files by running the following command:
```

Comment:
We wouldn't recommend that they install microshift-release-info on the MicroShift host. If it's installed at all, it would be on the image builder host or on some other host in the environment where they are setting up their build pipeline.

Reply:
@dhellmann Should this instruction be removed, or can this command be run as an alternative to the second one?

Reply:
You can keep it as an alternative, but there's no benefit to installing it on the MicroShift host. You have to have the contents of that RPM to build the image to boot the MicroShift host. So you could say "on the image builder host" instead (or whatever we call that host in the instructions for building images).